sk0-back/src/users/dtos/create-user.dto.ts

```diff
@@ -0,0 +1,25 @@
+import {
+  IsEmail,
+  IsString,
+  IsStrongPassword,
+  IsStrongPasswordOptions,
+} from 'class-validator';
+
+const passwordRules: IsStrongPasswordOptions = {
+  minLength: 6,
+  minUppercase: 1,
+  minLowercase: 1,
+  minNumbers: 1,
+  minSymbols: 0,
+};
+
+export class CreateUserDto {
+  @IsEmail()
+  email: string;
+
+  @IsStrongPassword(passwordRules)
+  password: string;
+
+  @IsString()
+  name: string;
+}
```

Comment:
Maybe we should add a minimum length here? Still not sure about the use cases. (Might definitely happen later)

Reply:
Thx changed for 3 now